test/e2e/framework/util.go

```diff
@@ -1327,6 +1328,8 @@ func ExpectNoError(err error, explain ...interface{}) {
 func ExpectNoErrorWithOffset(offset int, err error, explain ...interface{}) {
 	if err != nil {
 		e2elog.Logf("Unexpected error occurred: %v", err)
+		// Dump the stacktrace while an error occurs
```

Comment:
nit: I don't think the comments add any helpful information in either place. I'd just remove them.

Reply:
Deleted the comment